### ISSUE TITLE
Developer/contributing documentation?

### ISSUE THREAD

**Issue by victorb** [thomas-lowry/figma-plugin-ds#8]
Seems there is a lot of steps to build, minify and otherwise be able to reproduce the same files that exists here in this repository.

In order to contribute, it'd be great to be able to run these steps locally too. Currently I'm trying to figure out why the `<select>` element is not firing `change` events and was gonna use this websites examples as a base, but because I don't know/understand how this project is built, I cannot use it...

**Comment by jackiecorn** [comment]
https://github.com/thomas-lowry/figma-plugin-ds/pull/9/commits/ad7750e253b5e6c2cca09eb0656a17d64c32c91a
Check this pull request, it fixes the <select> issue.
Still not sure how to build it though.

**Comment by victorb** [comment]
Thanks a lot @jackiecorn, will give that a try! Found another issue that is now blocked by this one https://github.com/thomas-lowry/figma-plugin-ds/issues/10

**Comment by thomas-lowry** [comment]
Fixed in the latest release which is now available as an npm package with a Gulp 4 build.
Also please check out [Figma Plugin Boilerplate](https://github.com/thomas-lowry/figma-plugin-boilerplate) which comes pre-installed with this library. As a build process as well and is steup with CSS Purge to remove most unused styles to keep it light.
